src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Markup/Baml2006/WpfXamlType.cs

```diff
@@ -166,12 +166,12 @@ private XamlMember FindMember(string name, bool isAttached, bool skipReadOnlyChe
 
         protected override XamlMember LookupMember(string name, bool skipReadOnlyCheck)
         {
-            return FindMember(name, false /* isAttached */, skipReadOnlyCheck);
+            return FindMember(name, isAttached: false, skipReadOnlyCheck);
         }
 
         protected override XamlMember LookupAttachableMember(string name)
         {
-            return FindMember(name, true /* isAttached */, false /* skipReadOnlyCheck doens't matter for Attached */);
+            return FindMember(name, isAttached: true, false /* skipReadOnlyCheck doens't matter for Attached */);
```

Comment:
May have been skipped on purpose, if not: `skipReadOnlyCheck doens't matter for Attached` -> `skipReadOnlyCheck`